File: app/src/main/java/me/lukeforit/spaceofaday/ui/archive/items/EmptyApodItem.java
package me.lukeforit.spaceofaday.ui.archive.items;

import me.lukeforit.spaceofaday.R;

public class EmptyApodItem extends ArchiveItem {
    private String date;

    public EmptyApodItem(String date) {
        this.date = date;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    @Override
    public int getLayoutRes() {
        return R.layout.list_item_apod_empty;
    }
}

File: app/src/main/java/me/lukeforit/spaceofaday/ui/archive/items/EmptyApodItem.kt
package me.lukeforit.spaceofaday.ui.archive.items

import me.lukeforit.spaceofaday.R

class EmptyApodItem(listener: ArchiveItem.OnArchiveItemClickListener, var date: String?) : ArchiveItem(listener) {

    override val layoutRes: Int
        get() = R.layout.list_item_apod_empty
}
